tools/deploy.py

```diff
@@ -0,0 +1,490 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Is it all same as the file from mmdeploy official repo? If so, we'd better not to commit it into our repo, just advise users to clone one, and make mmdeploy path as an argument. We haven't copy files except configs from mmdet/mmpose before.

Reply:
Mostly the same as official repo. I change the import manner to adpat isort pipeline.